src/logger/POSTLoggerOptions.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import LogLevel from './LogLevel';
+
+export default interface POSTLoggerOptions {
+  /**
+   * URL to send logs.
+   */
+  url: string;
```

Comment:
no action required:
We can also take a `Request` object or `URL`, but we can start with `string` only and add more later without breaking changes.

Reply:
Sure thanks for the suggestion will track internally.